sklearn/datasets/openml.py

```diff
@@ -424,17 +424,34 @@ def _get_data_qualities(data_id, data_home):
         return None
 
 
-def _get_data_shape(data_qualities):
-    # Using the data_info dictionary from _get_data_info_by_name to extract
-    # the number of samples / features
+def _get_data_instances(data_qualities):
+    """
+    Retrieve the number of instances (samples) from data qualities
+
+    Parameters
+    ----------
+    data_qualities : list
```

Comment:
can't be a list. It's keyed by strings.

Reply:
Yes, it is a list of dict.
I updated the documentation accordingly.